dataactvalidator/config/sqlrules/b14_object_class_program_activity.sql

```diff
@@ -48,17 +49,20 @@ SELECT DISTINCT
         SUM(ussgl498100_upward_adjustm_cpe) +
         SUM(ussgl498200_upward_adjustm_cpe)
     ) + sf.amount AS "difference",
-    op.display_tas AS "uniqueid_TAS"
+    op.display_tas AS "uniqueid_TAS",
+    UPPER(op.disaster_emergency_fund_code) AS "uniqueid_DisasterEmergencyFundCode"
```

Comment:
Why are we displaying the `UPPER` version here? That's not what the user provided and everywhere else we display exactly what the user provided